<launch>

  <node name="mcu_interface" pkg="whipbot_v2" type="mcu_interface.py" output="screen"/>
  <node name="motion_generator" pkg="whipbot_v2" type="motion_generator_pos.py" output="screen"/>
  <node name="GUI_for_tuning" pkg="whipbot_v2" type="gui_for_PIDtuning.py" output="screen"/>

  <node name="wheel_odometry"   pkg="wheel_odometry" type="wheel_odometry_2wheel.py" output="screen">
    <param name="pulse_per_round" value="723.24"/>
    <param name="wheel_diameter" value="0.1524"/>
    <param name="tread" value="0.289"/>
  </node>

  <node name="joy_node"   pkg="joy" type="joy_node">
    <param name="autorepeat_rate" value="20"/>
    <param name="dev" value="/dev/input/js0"/>
  </node>

  <!-- <node pkg="cv_camera" name="cv_camera" type="cv_camera_node">
    <param name="device" type="string" value="/dev/video0"/>
    <param name="rate" value="10.0"/>
    <param name="image_width" value="640"/>
    <param name="image_height" value="480"/>
  </node> -->

  <node name="static_transform_publisher" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 map frame 100"/>

</launch>
